docs/UserGuide.md

```diff
@@ -15,20 +15,25 @@ interactions, FitBook is the perfect application for you to manage your clients!
 ## Quick Start
 
 1. Ensure you have Java `11` or above installed.
+    * Need help? Check out the [installation guide](https://docs.oracle.com/en/java/javase/11/install/overview-jdk-installation.html).
 
-2. Download the latest version of `FitBook.jar` from [here](https://github.com/AY2324S2-CS2103T-T17-3/tp/releases).
+1. Download the latest version of `FitBook.jar` from [here](https://github.com/AY2324S2-CS2103T-T17-3/tp/releases).
```

Comment:
it looks like the numbering is jumbled and everything is 1. is this the case for u?